package.json

```diff
@@ -52,6 +52,8 @@
   "dependencies": {
     "ajv": "^6.6.2",
     "clone-deep": "^4.0.1",
+    "js-yaml": "^3.12.2",
+    "json-schema-deref-sync": "^0.9.0",
     "swagger-parser": "^6.0.2"
```

Comment:
if you removed the dependency in `swagger-parser` please remove it from the dependencies as well